src/components/Poster/Poster.jsx

```diff
@@ -1,35 +1,38 @@
 import styles from "./Poster.module.css";
 
-function getColorClass(source, value, styles) {
-    let percentage    
-
-    if(source === "Internet Movie Database"){
-        percentage = parseFloat(value) / 10 * 100
-    }else if(source === "Rotten Tomatoes"){
-        percentage = parseInt(value, 10)
-    }else if(source === "Metacritic"){
-        percentage = parseInt(value, 10)
-    }
-
-    if(percentage >= 75)return styles.green
-    if(percentage >= 50)return styles.orange
-    return styles.red
+const getClassFromRatingValue = (src, value, styles) => {
```

Comment:
Move it to another file. Create a folder **utils** in **src/utils** and add **index.js** file for helper functions. Don't forget to export them.